SU2_CFD/src/solver_direct_mean_inc.cpp

```diff
@@ -3778,10 +3777,21 @@ void CIncEulerSolver::SetPrimitive_Limiter(CGeometry *geometry, CConfig *config)
   unsigned long iEdge, iPoint, jPoint;
   unsigned short iVar, iDim;
   su2double **Gradient_i, **Gradient_j, *Coord_i, *Coord_j,
-  *Primitive, *Primitive_i, *Primitive_j, *LocalMinPrimitive, *LocalMaxPrimitive,
-  *GlobalMinPrimitive, *GlobalMaxPrimitive,
+  *Primitive, *Primitive_i, *Primitive_j,
+  *LocalMinPrimitive = NULL, *LocalMaxPrimitive = NULL,
+  *GlobalMinPrimitive = NULL, *GlobalMaxPrimitive = NULL,
   dave, LimK, eps2, eps1, dm, dp, du, y, limiter;
   
+#ifdef CODI_REVERSE_TYPE
```

Comment:
No problem, thank you for the review. Would it make sense to move this method to CSolver? That is where the SolutionLimiter one is implemented.

Reply:
Not pressing, but we could consider this as part of a strategy to reuse more code btwn the comp. and density-based incomp. solver classes. Could be interesting to have an intermediate base class where the repeated routines live and then specialized routines down in the Euler/NS classes specific to each. But, this can wait for some other large refactoring to happen elsewhere first